contribs/gmf/src/directives/locationchooser.js

```diff
@@ -29,10 +29,12 @@ gmf.locationchooserDirective = function() {
     bindToController: true,
     controller: 'gmfLocationchooserController',
     controllerAs: 'ctrl',
-    template: '<select ' +
-        'ng-model="ctrl.selectedLocation" ' +
-        'ng-options="location.label for location in ::ctrl.locations" ' +
+    template:
+        '<select ng-model="ctrl.selectedLocation" ' +
         'ng-change="ctrl.setLocation()">' +
+        '<option value="{{location.extent}}"' +
+        'ng-repeat="location in ctrl.locations">' +
+        '{{location.label | translate}}</option>' +
```

Comment:
If we decide to use the `translate` directive in GMF directive partials then the `gmf` Angular module should depend on the `gettext` module.